lib/s3middleware/objectLegalHold.js

```diff
@@ -0,0 +1,111 @@
+const { parseString } = require('xml2js');
+const errors = require('../errors');
+
+/*
+    Format of the xml request:
+    <LegalHold>
+        <Status>ON|OFF</Status>
+    </LegalHold>
+*/
+
+/**
+ * validate status - validates status xml
+ * @param {object} status - parsed status object
```

Comment:
Based on the work in the method, I am not sure this is an object. It looks more like a string array with 1 item.

Reply:
added `[]`